glance/files/ocata/glance-glare.conf.Debian

```diff
@@ -1862,8 +1872,8 @@ swift_store_key = {{ server.storage.swift.store.key }}
 #
 #  (string value)
 #swift_store_config_file = <None>
-{% if server.storage.swift.store.config_file is defined %}
-swift_store_config_file = {{ server.storage.swift.store.config_file }}
+{% if server.storage.swift.store.references is defined %}
```

Comment:
can you touch to keep backward compatible, so sth like

if server.storage.swift.store.references is defined
   < new code >
else
  <keep the current>
end

Reply:
Hi @epcim, change is just reflecting same code block as in glance-api.conf https://github.com/salt-formulas/salt-formula-glance/blame/master/glance/files/ocata/glance-api.conf.Debian#L3221 commit https://github.com/salt-formulas/salt-formula-glance/commit/3a8678112ea5db62efa8e3d7a234ffbd3b7e303b this change was not done in glance-glare.conf which is now causing problems.
I think glance-api.conf and glance-glare.conf should have same code logic.
What do you think?
Thanks.

Reply:
@LotharKAtt @Martin819 Hi guys, could anyone of you comment or even merge this PR? Thanks a lot.